Example/example/SceneDelegate.swift

```diff
@@ -101,12 +101,24 @@ private extension SceneDelegate {
     @objc
     func cameraButtonTapped() {
         guard let waypoint = self.waypoints.randomElement() else { return }
-
+        
         let distance = CLLocationDistance.random(in: 10 ... 100_000)
-        self.viewController.mapView.camera = .init(lookingAtCenter: waypoint.coordinate,
-                                                   acrossDistance: distance,
-                                                   pitch: 0,
-                                                   heading: 0)
+        
+        let userAnchorPoint = self.viewController.mapView.userCourseView!.center
```

Comment:
What is the purpose of this change to the example app? Is it required that every implementor will need to have code like this to have the puck positioned in the bottom 1/3 of the map?

Reply:
No this has nothing to do with moving the puck down therefore it's not required by every implementor. 

This is a function in the sample project that randomly positions the camera for testing purposes. I wanted to test what happens when someone interferes with the camera during navigation - more specifically, what can go wrong if Maplibre-DSL (a SwiftUI wrapper) decides to reposition the camera. This allows to test this functionality while not embedded in SwiftUI.

Reply:
Ah ok, I never understood what this button was for. I'm still not sure that I do. Is it supposed to simulate a user interaction to drag the map around? Or are you trying to simulate some rogue software triggered camera movement? And did something about your changes to get the puck positioned in the bottom 1/3 of the screen cause this to behave differently?

In any case a comment on the method is worthwhile.